sound/soc/sof/intel/hda.h

```diff
@@ -487,6 +498,7 @@ void hda_dsp_dump_skl(struct snd_sof_dev *sdev, u32 flags);
 void hda_dsp_dump(struct snd_sof_dev *sdev, u32 flags);
 void hda_ipc_dump(struct snd_sof_dev *sdev);
 void hda_ipc_irq_dump(struct snd_sof_dev *sdev);
+void sof_hda_dsp_d0i3_work(struct work_struct *work);
```

Comment:
wouldn't it be better to also call this function `hda_dsp_d0i3_work()` for consistent naming?